app/models/user.rb

```diff
@@ -68,6 +69,10 @@ def active_ride
     Ride.where(driver_id: self.id).or(Ride.where(voter_id: self.id)).where(status: Ride.active_statuses).first
   end
 
+  def recent_complete_ride
+    self.rides.merge(Ride.completed).order('updated_at desc').first
```

Comment:
This is the user model and this merge appeared to be the way to incorporate
a Ride scope onto the rides of a user. Ride.completed is just a scope and
would return all completed rides?

On Mon, Aug 15, 2016 at 10:24 AM, John McGrath <notifications@github.com>
wrote:

> In app/models/user.rb
> <https://github.com/DevProgress/drive.vote/pull/64#discussion_r74797668>:
>
> > @@ -68,6 +69,10 @@ def active_ride
> >      Ride.where(driver_id: self.id).or(Ride.where(voter_id: self.id)).where(status: Ride.active_statuses).first
> >    end
> >
> > +  def recent_complete_ride
> > +    self.rides.merge(Ride.completed).order('updated_at desc').first
>
> This is probably my misunderstanding, by why the merge, rather than just
> calling 'Ride.completed'?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/DevProgress/drive.vote/pull/64/files/ddc41bfeefca580c93c6b12be6080d6e824f82ae#r74797668>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ATwgOEjHb9nvdszPWMUaYmVwmdN-2uIQks5qgKCzgaJpZM4Jj2K3>
> .
>